### DISCUSSION TITLE
Disable or Enable Module by code

### DISCUSSION THREAD

**Discussion by vitalybrandes** [OrchardCMS/OrchardCore#8523] [Q&A] [answered]
Can I or how can I disable or enable module by code?

Also, how can i check i some module is enabled?

**Comment by hishamco** [comment, marked as answer]
> Can I or how can I disable or enable module by code?

You could use `IShellFeaturesManager.EnableFeaturesAsync(features, force == true)` and `IShellFeaturesManager.DisableFeaturesAsync(features, force == true)`, you should supply the features to be enabled or disabled

> Also, how can i check i some module is enabled?

`IShellFeaturesManager.GetEnabledFeaturesAsync()`
